src/pile.js

```diff
@@ -1019,6 +1019,34 @@ const createPile = (
     setCover(newCover);
   };
 
+  const labelGraphics = new PIXI.Graphics();
+
+  const drawLabel = () => {
+    labelGraphics.clear();
+
+    const { width, height } = contentGraphics;
+
+    if (label.length) {
+      const labelWidth = width / label.length;
+      const labelHeight = 5;
+      label.forEach((color, index) => {
+        const labelX = labelWidth * index;
+        labelGraphics.beginFill(color, 1);
+        labelGraphics.drawRect(labelX, height, labelWidth, labelHeight);
```

Comment:
To avoid the offset issues we need to subtract the half-width and half-height of the pile from x and y.